.cursor/mcp.json

```diff
@@ -1,12 +1,5 @@
 {
   "mcpServers": {
-    "nx-mcp": {
```

Comment:
yes! normally it's no needed anymore if i understand correctly what said Jury. But maybe it a misunderstanding from my side :)